cypress/locators/menu/index.js

```diff
@@ -9,3 +16,8 @@ export const scrollBlock = () => cy.get(SUBMENU).find(SCROLL_BLOCK);
 export const lastSubmenuElement = () => submenuBlock().find("li div").last();
 export const menuDivider = () => cy.get(MENU_DIVIDER);
 export const segmentTitle = () => cy.get(SEGMENT_TITLE);
+export const onClickMenuItem = () =>
+  cy.get(MENU_ROOT).find("ul").first().find(`li:nth-child(5)`);
+export const submenuItem = () => cy.get(onClickMenuItem).find(SUBMENU_ITEM);
+export const noActionMenuItem = () =>
```

Comment:
we can delete this one locator and use `onClickMenuItem` but refactor it to something like:

`export const onClickMenuItem = (index) =>
   cy.get(MENU_ROOT).find("ul").first().find(`li:nth-child(index)`);`

but with changing the name to be it more understandable :)